src/components/steps/WelcomeForm.vue

```diff
@@ -1,68 +1,112 @@
 <template>
-  <Card class="welcome-card" :width="600" :height="440" previousText="Back" nextText="Start">
-    <template v-slot:header>
-      <div class="header">
-        Ey, listen!
+  <div>
+    <NavigationCard
+      title="Ey, listen!"
+      :previousStep="previousStep"
+      :nextStep="nextStep"
+      previousText="Back"
+      nextText="Let's do it"
+      v-if="sessionId"
+    >
+      <div class="container">
+        <p class="subtitle">
```

Comment:
You are repeating this in line 35. Could you create a component to avoid that repetition?